app/[lang]/error.tsx

```diff
@@ -22,30 +22,23 @@ export default function Error({
   }, [error])
 
   return (
-    <div className="flex h-full grow items-end justify-center">
+    <div className="flex h-screen flex-col justify-center">
```

Comment:
Looks like we lost <Topbar /> lets go ahead and add it back here as well lets make the error page have a linear gradient like the figma
``` suggestion
    <div className="flex h-screen flex-col justify-center bg-gradient-to-br from-[#0A1B2A] to-[#2A323C]">
    <Topbar />
```